v2/bundle/jwtbundle/bundle_test.go

```diff
@@ -134,11 +135,11 @@ func TestParse(t *testing.T) {
 		},
 		{
 			tf:  testFiles["non existent file"],
-			err: "unable to parse JWK Set: unexpected end of JSON input",
+			err: "jwtbundle: unable to parse JWKS: unexpected end of JSON input",
 		},
 		{
 			tf:  testFiles["missing kid"],
-			err: "error adding entry 1 of JWK Set: missing key ID",
+			err: "jwtbundle: error adding key 1 of JWKS: jwtbundle: keyID cannot be empty",
```

Comment:
Oh boy. The second `jwtbundle` there is a little gross. Side effect of creating a new error from an error produced by the error class. I'm talking with zeebo about a better pattern for this kind of thing. Until then, we might want to unwrap the error produced by `Add()` when passing it into the `jwtbundleErr.New()` method (i.e. via `errs.Unwrap()`). I'll leave a suggestion there.

Reply:
Yes, I thought about it but I was not sure about the best way to handle it. The suggested way makes sense for now.